src/components/CommandBar.js

```diff
@@ -102,37 +106,49 @@ const CommandBar = React.createClass({
 
   componentDidMount() {
     const shortcuts = this.context.shortcuts;
-    const handleEvent = (e, func) => {
+    const handlePressAnimation = button => {
+      if (button) {
+        button.style.opacity = "0";
+        button.style.transform = "scale(1.3)";
+        setTimeout(() => {
+          button.style.opacity = "1";
+          button.style.transform = "none";
+        }, 200);
+      }
+    };
+
+    const handleEvent = (e, func, button) => {
       e.preventDefault();
       e.stopPropagation();
+      handlePressAnimation(button);
       func();
     };
 
     COMMANDS.forEach((action) => shortcuts.on(
       getKey(action),
-      (_, e) => handleEvent(e, this.props[action]))
+      (_, e) => handleEvent(e, this.props[action], this[action]))
```

Comment:
I save a ref for each button on the component instance, so if an action is let's say 'stepOver',
this['stepOver'] is the corresponding button